<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickSearchDialog</class>
 <widget class="QDialog" name="QuickSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick Tag Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Mode:</string>
     </property>
     <property name="buddy">
      <cstring>modeCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Tag:</string>
     </property>
     <property name="buddy">
      <cstring>tagCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Value:</string>
     </property>
     <property name="buddy">
      <cstring>valueEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="valueEdit">
     <property name="placeholderText">
      <string>Value or Range like 2000-2300</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="optionBox">
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="tagCombo"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="modeCombo"/>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tagCombo</tabstop>
  <tabstop>valueEdit</tabstop>
  <tabstop>modeCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
